src/selected_words_counter/extract_files.py

```diff
@@ -86,19 +88,26 @@ def extract_msg_attachments(atarget_dir):
     # Extract all .msg files into a directory
     for afilepath in glob(atarget_dir + "*.msg"):
         afilepath = afilepath.replace("\\", "/")
-        try:
-            msg = extract_msg.Message(afilepath)
 
-            if len(msg.attachments) > 0:
-                a_output_directory = make_dir_from_filename(afilepath)
+        encodings = ["utf-8", "ISO-8859-1", "windows-1252"]
+        for encoding in encodings:
+            try:
+                msg = extract_msg.Message(afilepath, overrideEncoding=encoding)
 
-                for item in range(0, len(msg.attachments)):
-                    att = msg.attachments[item]
-                    msg.attachments[item].save(
-                        customPath=a_output_directory, customFilename=att.longFilename
-                    )
-        except Exception as e:
-            print(e)
+                if len(msg.attachments) > 0:
+                    a_output_directory = make_dir_from_filename(afilepath)
+
+                    for item in range(0, len(msg.attachments)):
```

Comment:
Shouldn't you check the encodings for the attachments as well?

Reply:
Don't think so because reading in the whole email should also encode? Not sure though worth to check out

Reply:
for now in a other pr